pkgs/tools/misc/quich/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "quich";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Usbac";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1n9c01q2v6g9wnmxp248yclhp8cxclnj0yyn1qrvjsn6srcpr22c";
+  };
+
+  doCheck = true;
+
+  makeFlags = [ "DESTDIR=" "PREFIX=$(out)" ];
```

```suggestion
  makeFlags = [ "DESTDIR=" "PREFIX=$(out)"  "CC=${stdenv.cc.targetPrefix}cc"  ];
```

Fixes build for me, macos 10.14